.github/workflows/python_dependency_tests.yml

```diff
@@ -38,6 +38,8 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.params.py_ver }}
+      - name: Install headers required by some python packages
```

Comment:
we might need to install this in the tox tests?

Reply:
yes, it is a gap in beam test coverage. this codepath might be covered by google-internal tests that use TFRecordIo.